Database/Entities/GameScore.cs

```diff
@@ -110,6 +107,18 @@ public class GameScore : UpdateableEntityBase, IProcessableEntity, IAdminNotable
     [Column("processing_status")]
     public ScoreProcessingStatus ProcessingStatus { get; set; }
 
+    /// <summary>
+    /// Id of the <see cref="Entities.Game" /> that the <see cref="GameScore" /> was set in
+    /// </summary>
+    [Column("game_id")]
+    public int GameId { get; set; }
+
+    /// <summary>
+    /// Id of the <see cref="Entities.Player" /> that set the <see cref="GameScore" />
+    /// </summary>
+    [Column("player_id")]
+    public int PlayerId { get; set; }
+
```

Comment:
Why move these down?

Reply:
I meant to actually put them above their navigations lol